studio/components/SalaryCsvInput/SalariesInput.tsx

```diff
@@ -0,0 +1,144 @@
+import { set, StringInputProps } from "sanity";
```

Comment:
I find the structure here difficult to understand. And I think it's just the naming and placing of functions and elements. 

Perhaps we can break up the structure/sort it a bit. 

We've got:
- a file-upload button
- a table to edit data from the file
- several different functions. 

I'm thinking we can make a clearer split to separate the two if possible? Perhaps something like this? 
<img width="744" alt="Screenshot 2024-09-05 at 11 09 06" src="https://github.com/user-attachments/assets/658bed5a-2343-4c12-96d5-373423b637fe">

so a folder structure like:
-salariesData
--salariesDataInput
--components
--salariesFileUploader
--salariesTableEditor

just quick random suggestions